modules/docs/arrow-docs/docs/docs/effects/fx/async/README.md

```diff
@@ -187,12 +187,13 @@ import arrow.effects.IO
 import arrow.unsafe
 import arrow.effects.extensions.io.unsafeRun.runBlocking
 import arrow.effects.extensions.io.fx.fxCancellable
+
+fun main() { // The edge of our world
 //sampleStart
-val (_, disposable) = fxCancellable {
-  !effect { println("BOOM!") }
-}
+  val (_, disposable) = fxCancellable {
```

Comment:
Is not possible to do destructuring outside local variables, so I moved this inside the main